### PR TITLE
SPCTR-2349 - Code refactor and optimisation 

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
Code optimisation remove unwanted codes

### CODE REVIEW

File: classes/class-uagb-init-blocks.php [hunk 1]
```diff
@@ -166,13 +153,10 @@ public function os_visibility( $block_attributes, $block_content ) {
 		);
 
 		$user_agent = isset( $_SERVER['HTTP_USER_AGENT'] ) ? sanitize_text_field( $_SERVER['HTTP_USER_AGENT'] ) : '';
-		
-		if ( isset( $os[ $value ] ) && preg_match( '@' . $os[ $value ] . '@', $user_agent ) ) {
-			return '';
-		}
 
-		return $block_content;
+		return preg_match( '@' . $os[ $block_attributes['UAGSystem'] ] . '@', $user_agent ) ? '' : $block_content;
```
Comment (RajkiranBagal) [inline classes/class-uagb-init-blocks.php:157]: equivalent of isset( $os[ $value ] ) check skipped?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]
@rahulvarma722 Added a comment on PR & also why there are changes in package-lock.json?

File: classes/class-uagb-init-blocks.php (hunk 1 above)
Comment (rahulvarma722) [reply classes/class-uagb-init-blocks.php:157]: i have add isset and isset added after my pr created

**Review by RajkiranBagal — APPROVED** [review]

**Review by patilvikasj — APPROVED** [review]

File: classes/class-uagb-post-assets.php [hunk 2]
```diff
@@ -248,8 +248,8 @@ public function prepare_widget_area_assets( $content ) {
 			return;
 		}
 
-		foreach ( $content as $key => $value ) {
-			if ( is_array( $value ) && isset( $value['content'] ) && has_blocks( $value['content'] ) ) {
+		foreach ( $content as $value ) {
+			if ( isset( $value['content'] ) && has_blocks( $value['content'] ) ) {
```
Comment (adi3890) [inline classes/class-uagb-post-assets.php:252]: ideally we should be checking is_array as it may gives issues in PHPStan

**Review by adi3890 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**